STT/CONTRIBUTING.md

```diff
@@ -19,7 +19,7 @@ $ docker build -f Dockerfile.train .
 ```
 
 Several parameters can be customized:
- - `ds_repo` to fetch DeepSpeech from a different repo than upstream
+ - `ds_repo` to fetch STT from a different repo than upstream
```

Comment:
nit: `ds_` should probably move to `stt_`

Reply:
Yep check out try two. Logs are coming though.